shuup/admin/toolbar.py

```diff
@@ -277,6 +277,23 @@ def render(self, request):
             yield '<li class="dropdown-header">%s</li>' % self.text
 
 
+class ProvideActionItem(DropdownItem):
```

Comment:
I probably wouldn't use `Provide` in the class name since we don't do that with provided form parts/sections. Maybe just `ActionItem` or `DropdownActionItem`.